docs/src/main/mdoc/jump_start_guide.md

```diff
@@ -192,8 +169,6 @@ If you call `traverse` instead of `map`, like `obj.traverse(fun)`, you'll get `G
 
 ```scala mdoc:silent
 import cats.syntax.traverse._
```

Comment:
I think we probably want to update all these examples to use the `all` syntax import